packages/Webkul/Admin/src/Resources/lang/en/app.php

```diff
@@ -811,6 +811,7 @@
                     'pending-ordered-qty'      => 'Pending Ordered Qty: :qty',
                     'pending-ordered-qty-info' => 'Pending Ordered quantity will be deducted from the respective inventory source after the shipment. In case of cancellation pending quantity will be available for sale.',
                     'title'                    => 'Inventories',
+                    'minimum-sellable-qty'     => 'Minimum purchase order quantity',
```

Comment:
This translation should be added for all local.

Reply:
Translation added